### DISCUSSION TITLE
Is it possible to ignore == for creating Anki cards

### DISCUSSION THREAD

**Discussion by DJackNelson** [reuseman/flashcards-obsidian#100] [Q&A]
Amazing plug in, thank you.

I like to use ==the highlight== markdown language.  When I go to make flashcards, it captures all my highlights too.  If I could switch to {cloze} only for cloze, that would be awesome.

**Comment by rossabc** [comment]
> Amazing plug in, thank you.
> 
> I like to use ==the highlight== markdown language. When I go to make flashcards, it captures all my highlights too. If I could switch to {cloze} only for cloze, that would be awesome.

Bumping this request, any possibility of adding the ability to switch off cloze for highlights?

**Reply by Bestboigill** [reply]
Have you guys found any workaround
